doc/classes/RemoteTransform.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RemoteTransform" inherits="Spatial" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		RemoteTransform mirrors the [Transform] of another Node in the scene.
 	</brief_description>
 	<description>
+		RemoteTransform mirrors the [Transform] of another Node (called the [code]remote node[/code]) in the scene.
+		It can be set to track another Node's position, rotation, scale or all of those and update its own accordingly, using either global or local coordinates.
+		The [code]remote node[/code] has to inherit from [Spatial] (i.e. it actually has to have a [Transform] to mirror).
+		Note that while the name may suggest otherwise, RemoteTransform has nothing to do with networking. It only works on local Nodes.
+		For a 2D version, see [RemoteTransform2D].
```

Comment:
We should only do this if we're going to be consistent and link every 2D <-> 3D equivalent. Seems like too much.